src/main/java/uk/gov/hmcts/darts/common/repository/TranscriptionRepository.java

```diff
@@ -39,4 +40,20 @@ List<TranscriptionEntity> findAllByTranscriptionStatusNotInWithCreatedDateTimeBe
         """
     )
     List<TranscriptionEntity> findByHearingId(Integer hearingId);
+
+    @Query("""
+        SELECT t
+        FROM TranscriptionEntity t, HearingEntity hearing
+        WHERE hearing.id = :hearingId
+        AND t.hearing = hearing
+        AND t.transcriptionType = :transcriptionType
+        AND t.startTime = :startTime
```

Comment:
I'm wondering if this query copes with the startTime or endTime being NULL?
Thinking we might need to tweak the condition to check `IS NULL`

Reply:
@davet1985 could you write a test to cover Richards scenario and see what happens? In fact, looks like you might need to write some unit tests to cover all scenarios?